src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { updateArticleVotes } from "../api"
+
+const ArticleVotes = ({ article_id, votes_count }) => {
+    const [userVotes, setUserVotes] = useState(0);
+    const [isError, setIsError] = useState(false)
+
+    const handleLikeClick = (event) => {
+        setIsError(false)
+        event.preventDefault();
+
+        setUserVotes((currentUserVote) => {
+            return currentUserVote + 1;
+        })
+
+        updateArticleVotes(article_id, 1)
+            .catch((error) => {
+                setIsError(true)
+                setUserVotes((currentUserVote) => {
+                    return currentUserVote - 1;
+                })
+            })
+
+    }
+
+    const handleUnlikeClick = (event) => {
+        setIsError(false)
+        event.preventDefault();
+
+        setUserVotes((currentUserVote) => {
+            return currentUserVote - 1;
+        })
+        updateArticleVotes(article_id, -1)
+            .catch((error) => {
+                setIsError(true)
+                setUserVotes((currentUserVote) => {
+                    return currentUserVote + 1;
+                })
+            })
+    }
+
+    return (
+        <div>
```

Comment:
Could we use anything better than a div here?

Reply:
I used section